### ISSUE TITLE
really weird visual bug with 2012 & 2013M-2017 layout + title bug on 2008-2011 + channel title bug on2016

### ISSUE THREAD

**Issue by zaxel-github** [lightbeam24/CustomTube#127]
its probably a RYD issue (somehow because i have compat disabled), but the bell and subscriber count are very misaligned. i use opera gx and i have reformat like counter disabled.

![Screenshot 2023-11-13 113234 EDIT](https://github.com/lightbeam24/CustomTube/assets/79190563/a3dec9f4-9cee-4dd9-ac6b-29ef2635bed0)
2016 (with channel title bug)
![Screenshot 2023-11-13 113828 EDIT](https://github.com/lightbeam24/CustomTube/assets/79190563/97b77d57-79fe-4f5d-a972-042feb0bf166)
2012
![Screenshot 2023-11-13 113940](https://github.com/lightbeam24/CustomTube/assets/79190563/16b95eac-ea49-408c-a0d0-eb0246b3bf81)
2013M
![Screenshot 2023-11-13 114016](https://github.com/lightbeam24/CustomTube/assets/79190563/726c85f6-58cb-41f7-ac20-e6d6208b6656)
2014/2015?
![Screenshot 2023-11-13 114341](https://github.com/lightbeam24/CustomTube/assets/79190563/309448ae-bcaf-4605-aa2e-cc07e417f156)
2017

the only one i can confirm does not have any issues are 2008-2011 & 2013E, and 2019 & up. 2008-2011 has its own issue though.

+

while seeing if the bug happened on layouts pre cosmic panda, grey lines on title appeared. here are screenshots.
![Screenshot 2023-11-13 114117](https://github.com/lightbeam24/CustomTube/assets/79190563/a8607bd9-187c-488b-9d3c-87fd9e645617)
2011
![Screenshot 2023-11-13 114142](https://github.com/lightbeam24/CustomTube/assets/79190563/20fefffa-9283-482f-8528-76ebab5e37ef)
2009L
![Screenshot 2023-11-13 114224](https://github.com/lightbeam24/CustomTube/assets/79190563/b3f354da-6bf0-4a58-b14d-414b8f381588)
2008

**Comment by lightbeam24** [comment]
Don't have much time right now, but can you see if some of this stuff happens in Firefox? I can't reproduce the title scrollbar thing. Also, enable "Hide Join button" in Tweaks.

**Comment by zaxel-github** [comment]
yes, it does happen in firefox aswell, looks even worse in some cases, also another bug

![Screenshot 2023-11-14 155027](https://github.com/lightbeam24/CustomTube/assets/79190563/28141cbe-3fb4-47d3-a4bc-70f6a3218d7f)

2017

ok, now back to this

![Screenshot 2023-11-14 155326](https://github.com/lightbeam24/CustomTube/assets/79190563/c79b034a-5e64-450d-9864-83206ca1b869)

2017 (some of it is just ambient mode, which i did disable in later screenshots)

![Screenshot 2023-11-14 155431](https://github.com/lightbeam24/CustomTube/assets/79190563/a3140be1-ea1c-4a23-a228-bf3865691536)

2013M (fixed with firefox when unsubbed)

![Screenshot 2023-11-14 155519](https://github.com/lightbeam24/CustomTube/assets/79190563/123711ed-8e9d-48de-be5e-a33d60fee003)
 
2013L (same!)

![image](https://github.com/lightbeam24/CustomTube/assets/79190563/b80af04b-683f-44fa-811a-0a7b77515081)

2016 (subbed)

![Screenshot 2023-11-14 160134](https://github.com/lightbeam24/CustomTube/assets/79190563/8e0027aa-addd-46cb-8c9b-25b433487c0c)

2013L? (fixed with firefox!)

![Screenshot 2023-11-14 160156](https://github.com/lightbeam24/CustomTube/assets/79190563/46190ef3-e149-4540-91ef-aefdb54dcdaf)

2014 (subbed)

so maybe its windows or something else, because some have been fixed with firefox and some havent?

oh, and the title bug was fixed in firefox, channel title bug too.
